src/applications/edu-benefits/1995-STEM/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "22-1995-STEM Education benefits form",
+  "entryFile": "./edu-benefits-entry.jsx",
+  "entryName": "1995-stem-edu-benefits",
```

Comment:
I think `stem` is capitalized in the page in `vagov-content`, those should probably match exactly.